File: app/src/main/java/ims/yang/com/ims/BootCompleteReceiver.java
package ims.yang.com.ims;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.widget.Toast;
import ims.yang.com.ims.util.MyToast;

public class BootCompleteReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        MyToast.INSTANCE.toastLong(context,"Boot Complete");
    }
}

File: app/src/main/java/ims/yang/com/ims/BootCompleteReceiver.kt
package ims.yang.com.ims

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.widget.Toast
import ims.yang.com.ims.util.MyToast

class BootCompleteReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        MyToast.toastLong(context, "Boot Complete")
    }
}
